alpha/action/migrate_test.go

```diff
@@ -126,6 +182,10 @@ func TestMigrate(t *testing.T) {
 				require.Equal(t, expectedData, string(actualData))
 				return nil
 			})
+
+			if s.migrationCount != 0 {
+				require.Equal(t, s.migrationCount, migrationPre+s.migrationCount)
```

Comment:
I very well could be misunderstanding this code, but this will fail if `migrationPre` is ever non-zero, right?

And the only reason it doesn't fail now is because there's only one spec that increments the counter?

I'm having trouble conjuring an approach that remains simple and overcomes this problem, but I'll clone and tinker later.

Reply:
Yes, this does an explicit compare against the counter value in the test spec... which will only work when the count of TCs attempting this feat is one.  
It should be verifying that the count went up by the amount in the test spec, instead:
```golang
require.Equal(t, migrationCounter, migrationPre+s.migrationCount)
```